packages/PoppyPrint-Core.package/PPFormatter.class/instance/isTypeOfNewLineMarker..st

```diff
@@ -0,0 +1,4 @@
+helper
+isTypeOfNewLineMarker: aNode
+
+	^ ((self isOptionalEmptyLineMarker: aNode) or: (self isOptionalNewLineMarker: aNode) or: (self isEmptyLineMarker: aNode))
```

Comment:
Same here
- `statement1 or: [statement2]` not `statement1 or: statement2`
- `statement1 and: [statement2]` not `statement1 and: statement2`